lib/portlayer/network/context.go

```diff
@@ -892,53 +979,11 @@ func (c *Context) UnbindContainer(h *exec.Handle) ([]*Endpoint, error) {
 		// clear out assigned ip
```

Comment:
Worth noting that this is the reason we cannot combine UnbindContainer and UnbindContainerById further